collector/src/main.rs

```diff
@@ -585,6 +664,37 @@ fn main_result() -> anyhow::Result<i32> {
             (@arg RUSTDOC: --rustdoc +takes_value "The path to the local rustdoc to benchmark")
         )
 
+        (@subcommand diff_local =>
+            (about: "Profiles and compares two toolchains with one of several profilers")
+
+            // Mandatory arguments
+            (@arg PROFILER: +required +takes_value
+             "One of: 'self-profile', 'time-passes', 'perf-record',\n\
+             'oprofile', 'cachegrind', 'callgrind', 'dhat', 'massif',\n\
+             'eprintln', 'llvm-lines'")
+            (@arg RUSTC1:   +required +takes_value "The path to the local rustc to benchmark")
+            (@arg ID1:      +required +takes_value "Identifier to associate benchmark results with")
+            (@arg RUSTC2:   +required +takes_value "The path to the local rustc to benchmark")
+            (@arg ID2:      +required +takes_value "Identifier to associate benchmark results with")
+
+             // Options
+            (@arg BUILDS: --builds       +takes_value
+             "One or more (comma-separated) of: 'Check', \n\
+             'Debug', 'Doc', 'Opt', 'All'")
+            (@arg CARGO:   --cargo       +takes_value "The path to the local Cargo to use")
+            (@arg EXCLUDE: --exclude     +takes_value
+             "Exclude all benchmarks matching anything in\n\
+             this comma-separated list of patterns")
+            (@arg INCLUDE: --include     +takes_value
+             "Include only benchmarks matching something in\n\
+             this comma-separated list of patterns")
+            (@arg OUT_DIR: --("out-dir") +takes_value "Output directory")
+            (@arg RUNS:    --runs        +takes_value
+             "One or more (comma-separated) of: 'Full',\n\
+             'IncrFull', 'IncrUnchanged', 'IncrPatched', 'All'")
+            (@arg RUSTDOC: --rustdoc +takes_value "The path to the local rustdoc to benchmark")
```

Comment:
Hm, I'd expect us to pull this from the same place we do rustc... basically, take the rustc path and ../rustdoc